gui/src/main/java/org/jboss/wise/gui/ResponseLogHandler.java

```diff
@@ -83,7 +85,8 @@ private void doLog(LogicalMessageContext smc) {
 	    } catch (Exception e) {
 		PrintWriter ps = new PrintWriter(outputStream);
 		ps.println("Exception getting response message log: ");
-		e.printStackTrace(ps);
+		//debug only  e.printStackTrace(ps);
+			log.error(e);
```

Comment:
The e.printStackTrace(ps) here was actually printing the stacktrace on the provided outputStream; by printing it to the logger you're actually breaking this functionality.